### DISCUSSION TITLE
七月更新

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#1290] [通知]
## 📣 高亮功能

1. 适配 Xbox 手柄操作，界面针对 TV 进行了优化
2. 完全支持繁体中文 （感谢 @jay900604 的贡献），包括对视频标题/评论/弹幕等内容的自动转译

## 🤣 已知问题

1. Xbox 部分界面，如播放界面和设置界面，均保留了光标操作
2. Xbox 视频清晰度过高（如 4K）有概率造成闪退问题，目前不确定是软件引起的还是设备引起的，有待进一步排查
3. Xbox 上关闭弹幕有助于提高播放的稳定性和帧率

## 更改记录

- 以代理替漫游 @Richasy (#1255)
    为避免不必要的误会，哔哩内部不再以漫游称呼番剧代理功能，并明确写明哔哩不支持现有的哔哩漫游服务。如何构建适用于哔哩的基础反向代理，请参考项目 Wiki

## 🚀 功能

- 添加视频收藏夹和用户空间页面 @Richasy (#1283)
- 适配 XBOX @Richasy (#1280)
- 修正繁體中文語言的用詞 @jay900604 (#1272)
- 提供对繁体中文的完全支持 @Richasy (#1265)

## 🐛 Bug 修复

- 修复xbox上进度条的状态切换问题 @Richasy (#1284)
- 修复默认收藏夹为空时的空引用错误 @Richasy (#1282)
- 修复因查询P2P失败造成的播放问题 @Richasy (#1281)
- 修复因调整音量造成的UI卡死 @Richasy (#1271)
- 在全窗口/全屏时去除播放器圆角 @Richasy (#1270)
- 添加评论刷新按钮以应对请求失败的情况 @Richasy (#1269)
- 修复部分视频打开后弹幕不可见的情况 @Richasy (#1268)
- 修复动态没有通知的问题 @Richasy (#1260)
- 修改跳进图标 @Richasy (#1259)
- 修复关注按钮没有正确执行的问题 @Richasy (#1258)
- 双击改变播放模式后，恢复播放 @Richasy (#1257)
- 修复触屏下的双击行为 @Richasy (#1247)
- 修复动态及消息条目的显示问题 @Richasy (#1244)

## 🧰 管理与维护

- 清理代码引用及更新依赖 @Richasy (#1285)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2207.1.0'>七月更新</a>.</em>

**Comment by amtoaer** [comment]
这个版本下载安装后会提示更新，明明已经是最新了..不清楚和release对应的tag是`v2.2206.4`有没有关系.. 😢 
![image](https://user-images.githubusercontent.com/32017007/176573584-a60f54ee-b5d4-4b70-a65e-ce37468864b5.png)
![image](https://user-images.githubusercontent.com/32017007/176573599-87da7083-24eb-49b7-a7a5-e2fa1b816c04.png)
![image](https://user-images.githubusercontent.com/32017007/176573839-08881363-7537-47d9-a2b3-eaa7de730cad.png)
还有想问下大佬这个版本会发布到微软商店嘛？

**Reply by Richasy** [reply]
嘶……不好意思忘了改了，现在应该好了。
会发商店，但是商店要审核，我的包已经提交了，不知道今天能不能发布

**Reply by maxlen727** [reply]
音量卡死问题终于修了，没修的时候简直是不能用啊

**Comment by CsyBarry** [comment]
最新版本怎么忘了数字签名啊

**Reply by Richasy** [reply]
你指的是？

**Comment by HumanMus1c** [comment]
问题1：视频播放时弹幕一顿一顿的。
问题2：搜索没有历史记录。
问题3：搜索不能复制当前搜索的内容，也不能修改搜索内容。

**Comment by HumanMus1c** [comment]
建议1：希望播放控制可以重新设计，太难用了

**Comment by chong527** [comment]
請問能否看到影片的彈幕列表？ 如果沒有可以加入這個功能嗎？

還有全螢幕的時候開啟其他程式時，bilibili會自動最小化，可以關閉這個功能嗎？

**Comment by HaoZhou185** [comment]
IT之家来的，顶一下大佬顺便star一下
